emma-language/src/main/scala/org/emmalanguage/compiler/lang/cf/CFG.scala

```diff
@@ -100,39 +57,57 @@ private[compiler] trait CFG extends Common {
      * - All containment relations from LHS to RHS are contained in the graph.
```

Comment:
Could you update these comments to match them to the three fields above? (I guess containment is the same as nesting?)